src/main/java/io/inscopemetrics/kairosdb/HistogramModule.java

```diff
@@ -59,7 +59,11 @@
 import javax.inject.Provider;
 
 /**
- * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+ * Guice injection module binds instances for plugin.
+ *
+ * TODO(ville): Rename this module.  This is a breaking change.
```

Comment:
Did you intend to rename before merging?

Reply:
No, it's a non-backwards compatible change. It's a note for the future and there is a Github issue for it too; #29.